### PR TITLE
CRI image cache config

### PR DESCRIPTION
**Pull request by koct9i**
- Fix detecting qota for volume without storage requests
  
- Add CRI image cache config

### CODE REVIEW

File: pkg/ytconfig/canondata/TestGetExecNodeConfigWithCri/single-container-without-job-resources/test.canondata [hunk 1]
```diff
@@ -127,6 +127,9 @@
                     namespace=yt;
                     "base_cgroup"="/yt";
                 };
+                "cri_image_cache"={
+                    capacity=4294967296;
+                };
```
Comment (l0kix2) [inline pkg/ytconfig/canondata/TestGetExecNodeConfigWithCri/single-container-without-job-resources/test.canondata:132]: Why canondata config is changed if tests are not? Is it some default value (I don't see value in diff)?
Is it expected that config would change if no new fileds in spec is specified?

Comment (koct9i) [reply pkg/ytconfig/canondata/TestGetExecNodeConfigWithCri/single-container-without-job-resources/test.canondata:132]: cri image cache section is added for cri env setip

Comment (koct9i) [reply pkg/ytconfig/canondata/TestGetExecNodeConfigWithCri/single-container-without-job-resources/test.canondata:132]: I've added 4gb quota for image-cache location in previous pr

Comment (l0kix2) [reply pkg/ytconfig/canondata/TestGetExecNodeConfigWithCri/single-container-without-job-resources/test.canondata:132]: i see, so if nothing new (cache quota) would be specified in real spec, we won't have a diff in configs?

Comment (koct9i) [reply pkg/ytconfig/canondata/TestGetExecNodeConfigWithCri/single-container-without-job-resources/test.canondata:132]: without explicit quota there will be volume storage request.

**Review by l0kix2 — APPROVED** [review]

**Pull request merged**